mars/deploy/oscar/base_config.yml

```diff
@@ -36,3 +36,6 @@ scheduling:
     max_workers: 100
     scheduler_backlog_timeout: 60
     worker_idle_timeout: 120
+global_config:
```

Comment:
For me, there is no need to add a separate section called `global_config` as the name is somehow confusing. Just put `metrics` as a root sector will look better for me.